### PR TITLE
Implement Card component

### PR DESCRIPTION
**Pull request by omardeleo**
Add some basic Card markup and styling, with placeholder buttons for EDIT and VOTE.

### CODE REVIEW

File: client/src/components/cards/CardsIndex.jsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import Form from './Form';
-import * as cardsActions from '../../actions/cards';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { withStyles } from '@material-ui/core/styles';
```
Comment (BlaneCordes) [inline client/src/components/cards/CardsIndex.jsx]: we should probably setup an scss directory and start including our stylesheets and libraries in there, import them on the index page

File: client/src/components/cards/CardsIndex.jsx [hunk 2]
```diff
@@ -16,13 +30,29 @@ class CardsIndex extends Component {
   }
 
   render() {
+    const { classes } = this.props;
```
Comment (BlaneCordes) [inline client/src/components/cards/CardsIndex.jsx]: whats the reason for the { }

File: client/src/components/cards/CardsIndex.jsx [hunk 3]
```diff
@@ -16,13 +30,29 @@ class CardsIndex extends Component {
...
+                <Button>
+                  EDIT
+                </Button>
+                <Button>
+                  VOTE
+                </Button>
+              </CardActions>
+            </Card>
+          )
         })}
```
Comment (BlaneCordes) [inline client/src/components/cards/CardsIndex.jsx]: you can do .bind(this) to maintain the scope and you wont need to set that constant up top

File: client/src/components/cards/CardsIndex.jsx [hunk 4]
```diff
@@ -44,4 +74,4 @@ function mapDispatchToProps(dispatch) {
   }
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(CardsIndex);
+export default connect(mapStateToProps, mapDispatchToProps)(withStyles(styles)(CardsIndex));
```
Comment (BlaneCordes) [inline client/src/components/cards/CardsIndex.jsx]: is this necessary? ```withStyles(styles)```

File: client/src/components/cards/CardsIndex.jsx (hunk 1 above)
Comment (omardeleo) [reply client/src/components/cards/CardsIndex.jsx]: Ok, I'll create another issue for SCSS. For now I'm adding some basic styling to our material components, following the examples in the material design documentation and examples. 
[Card component demo](https://material-ui.com/demos/cards/).

File: client/src/components/cards/CardsIndex.jsx (hunk 2 above)
Comment (omardeleo) [reply client/src/components/cards/CardsIndex.jsx]: {} is destructuring props to access the classes passed by withStyles higher order component.

File: client/src/components/cards/CardsIndex.jsx (hunk 4 above)
Comment (omardeleo) [reply client/src/components/cards/CardsIndex.jsx]: withStyles is a higher order component used to inject CSS into the DOM. It appears to be the recommended method to style components in material design.

**Comment by omardeleo** [comment]
@BlaneCordes Thanks for the review and feeback!

**Pull request merged**